### PR TITLE
Dont fwd azure emulator & sql to host in compose

### PR DESCRIPTION
**Pull request by KnicKnic**
## Description
Do not expose azure emulator and sql ports to host. This will prevent them from colliding if sql server or azure storage emulator is installed in the host already

## Testing
Did the following simultaneously 
* Started azure storage emulator and sql server on machine 
* ran dicom tests
* started docker-compose environment
* uploaded dicom files to docker-compose environment

### CODE REVIEW

File: samples/docker/docker-compose.yaml [hunk 1]
```diff
@@ -21,18 +21,18 @@ services:
     container_name: azurite
     image: mcr.microsoft.com/azure-storage/azurite
     restart: always
-    ports:
-      - "10001:10001"
-      - "10000:10000"
+    # ports:
```
Comment (brandonpollett) [inline samples/docker/docker-compose.yaml:24]: nit: maybe add a comment around explaining to un-comment these if you want connect directly to these services from the host.

**Review by brandonpollett — APPROVED** [review]

File: samples/docker/docker-compose.yaml (hunk 1 above)
Comment (KnicKnic) [reply samples/docker/docker-compose.yaml:25]: Done

**Pull request merged**